sentry-android-core/build.gradle.kts

```diff
@@ -56,6 +56,8 @@ android {
 
         // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks.
         isCheckReleaseBuilds = false
+
+        disable.add("UnknownNullness")
```

Comment:
@philipphofmann help me to fix those
so no manoelly linter :)

Reply:
LOL

Reply:
basic all public getters and setters would need either @NotNull or @Nullable, overwhelming hehe, I'd guess around 700

Reply:
I just added this disable rule to make CI happier and focus on the other lint issues.

Reply:
![image](https://user-images.githubusercontent.com/1633368/103030068-0da7ab80-4529-11eb-8994-cd4cf021f636.png)